spec/factories/session.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  # Start the sequence at the current year. Note that every call to create a new session will have a new year.
+  sequence(:year, Time.now.year)
+
+  factory :session do
+    pay { 20.00 }
+    year # Set it to the sequence
+
+    trait :fall do
+      start_date { Time.new(year, 9) }
```

Comment:
```suggestion
      start_date { Time.new(year, 9, 1) }
```

Add days to start_dates for consistency